x-pack/plugin/deprecation/src/test/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecksTests.java

```diff
@@ -2183,6 +2207,37 @@ public void testZenDiscoverySettings() {
         }
     }
 
+    public void testDynamicSettings() {
+        Settings clusterSettings = Settings.builder()
+            .put(ElectMasterService.DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTING.getKey(), randomInt())
+            .build();
+        Settings nodettings = Settings.builder().build();
+        final PluginsAndModules pluginsAndModules = new PluginsAndModules(Collections.emptyList(), Collections.emptyList());
+        final XPackLicenseState licenseState = new XPackLicenseState(Settings.EMPTY, () -> 0);
+        Metadata metadata = Metadata.builder().persistentSettings(clusterSettings).build();
+        ;
```

Comment:
?

Reply:
I'd apparently typed two semicolons at the end of the previous line, and spotlessApply fixed that for me. :)